CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v8.5.0](https://github.com/voxpupuli/puppet-rabbitmq/tree/v8.5.0) (2019-01-15)
```

Comment:
Believe this should be updated by whomever does the release and not part of a pull request.

Reply:
Or to put it differently, I did the normal process, I think (https://voxpupuli.org/docs/#releasing-a-new-version-of-a-module)
Seems to track w/ the way the version looks in the changelog in https://github.com/voxpupuli/puppet-rabbitmq/pull/746/files